gateway/scim_ethermab.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2023, Cisco Systems, Inc. and/or its affiliates.
+# All rights reserved.
+# See LICENSE file in this distribution.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module implements Ethernet MAB dispatch for SCIM.
+"""
+
+import importlib.util
+
+from models import EtherMABExtension
+from database import session
+from config import ISE_SUPPORT, ISE_HOST, ISE_USERNAME, ISE_PASSWORD
+from tiedie_exceptions import SchemaError,DeviceExists
+
+if importlib.util.find_spec("ciscoisesdk"):
```

Comment:
Why do we need this?

Reply:
If ciscoisesdk is not installed, I don't want the whole thing to blow up.

Reply:
It's included in the requirements.txt though?